src/python-exec.c

```diff
@@ -140,8 +140,16 @@ int resolve_symlinks(char* outbuf, const char* path)
 
 		sys_path = getenv("PATH");
 		/* mimic exec*p() behavior */
-		if (!sys_path)
-			sys_path = FALLBACK_PATH;
+		if (!sys_path) {
+			if (strlen(FALLBACK_PATH) == 0) {
```

Comment:
Why would FALLBACK_PATH ever be empty? This looks like a broken install.

Reply:
Couldn't tell you. Here's the ebuild for 2.4.4 that builds this without `--with-fallback-path`: https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-lang/python-exec/python-exec-2.4.4.ebuild

The 2.4.5 ebuild does set `--with-fallback-path`: https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-lang/python-exec/python-exec-2.4.5.ebuild

Reply:
FYI my tests are without `--with-fallback-path` set during configure.

Reply:
That's because the fallback code has been added in 2.4.5. We need to stabilize this version.